releases/0.3/upgrading/grid-v0.3.1-from-v0.2.1.md

```diff
@@ -0,0 +1,32 @@
+# Upgrading to Grid v0.3.1 from Grid v0.2.1
+
+<!--
+  Copyright 2018-2022 Cargill Incorporated
+  Licensed under Creative Commons Attribution 4.0 International License
+  https://creativecommons.org/licenses/by/4.0/
+-->
+
+## Changes
+
+- [Smart Contract Updates](#smart-contract-updates)
+- [System Keygen Command](#system-keygen-commands)
+
+## Smart Contract Updates
+
+Grid Pike has been incremented to version 3. This version made some fundamental
+changes to Pike data models and changed the addressing scheme to fit the pattern
+of the rest of the Grid smart contracts. Due to the fundamental nature of the
+changes, Pike v2 does not support backwards compatibility and Pike org
```

Comment:
We could probably remove the first/second sentence here and update the next sentence to remove that beginning part about the "fundamental changes" and could probably keep the second half of the sentence. Then we can just say "Pike v2 (which it still is Pike v2 technically since we haven't updated it for this release, right?) does not support ..."  rather than that beginning part of the paragraph

Reply:
must've missed a save. Rewritten to be for PO 0.3